linux_sql/README.md

```diff
@@ -1,2 +1,74 @@
 # Linux Cluster Monitoring Agent
```

Comment:
instead of opening a new PR, you should have used the previous PR and resolve all comments

Reply:
ok, i will resolve these comments